### PR TITLE
[IMP] invoice_discount: Migrating to 11.0

### PR DESCRIPTION
**Pull request by ebirbe**
Migrating module to Odoo v11.0

* Using new API on models.
* Removing deprecated sale_config_settigs configuration.
* Using standard file and folder names.
* Removing dependency to `sale.group_discount_per_so_line`, now using `base.group_no_one` as Odoo does for `discount` field in views.

### CODE REVIEW

**Comment by ebirbe** [comment]
Hello @moylop260 I need this module active in v11 to complete the task described on https://git.vauxoo.com/vauxoo/lasec/issues/1

Could you review, please?

/cc @rdmachadog

File: invoice_discount/__manifest__.py [hunk 1]
```diff
@@ -32,17 +32,15 @@
     "license": "AGPL-3",
     "depends": [
         "account",
-        "sale"
     ],
     "demo": [],
     "data": [
-        "view/invoice_view.xml",
-        "view/config_settings_action.xml"
+        "views/invoice_view.xml",
```
Comment (moylop260) [inline invoice_discount/__manifest__.py]: bumpversion to 11.0...

File: invoice_discount/models/__init__.py [hunk 2]
```diff
@@ -23,5 +23,4 @@
 #    along with this program.  If not, see <http://www.gnu.org/licenses/>.
```
Comment (moylop260) [inline invoice_discount/models/__init__.py]: Use small comments from template and remove utf-8 for py3 is not required

File: invoice_discount/views/invoice_view.xml [hunk 3]
```diff
@@ -1,42 +1,40 @@
 <?xml version="1.0" encoding="utf-8"?>
-<openerp>
-    <data>
-        
+<odoo>
+
```
Comment (moylop260) [inline invoice_discount/views/invoice_view.xml]: remove extra spaces
Use 4 spaces for each child

**Review by moylop260 — CHANGES_REQUESTED** [review]

**Comment by ebirbe** [comment]
@moylop260 Review corrections done at last commit. Could you review again and merge if possible, please?

**Comment by ebirbe** [comment]
Hello @moylop260 is something blocking this merge?

**Pull request merged**